src/baseline.rs

```diff
@@ -259,30 +297,8 @@ impl BaselineLoader for RegistryBaseline {
         // - Most likely the user cares about the last official release
```

Comment:
This comment block seems relevant to the code block you extracted into a function. Probably move it there too?